src/prefs/ApplicationPrefs.cpp

```diff
@@ -68,12 +69,44 @@ void ApplicationPrefs::PopulateOrExchange(ShuttleGui & S)
    S.SetBorder(2);
    S.StartScroller();
 
-   S.StartStatic(XO("Update Audacity"));
+   /* i18n-hint: Title for the update notifications panel in the preferences dialog. */
+   S.StartStatic(XO("Update notifications"));
    {
-      S.TieCheckBox(
-          XO("&Check for Updates...").Stripped(TranslatableString::Ellipses | TranslatableString::MenuCodes),
-          DefaultUpdatesCheckingFlag);
+      /* i18n-hint: Check-box title that configures periodic updates checking. */
+      S.TieCheckBox(XC("Check for updates", "application preferences"), DefaultUpdatesCheckingFlag);
+
+      S.StartVerticalLay();
+      {
+         S.AddFixedText(XO(
+            "App update checking requires network access. In order to protect your privacy, Audacity does not store any personal information."),
+            false, 470);
+
+         S.StartHorizontalLay(wxALIGN_LEFT);
+         {
+            S.SetBorder(0);
+
+            S.AddSpace(2, 0);
+
+            /* i18n-hint: The first part of "See our Privacy Policy for more info". Trailing space is required. */
+            S.AddFixedText(XO("See "));
+
+            S.AddWindow(
+               safenew wxHyperlinkCtrl(
+                  /* i18n-hint: The second part of "See our Privacy Policy for more info". */
+                  S.GetParent(), wxID_ANY, XO("our Privacy Policy").Translation(),
+                  "https://www.audacityteam.org/about/desktop-privacy-notice/"));
+
+            /* i18n-hint: The last part of "See our Privacy Policy for more info". Leading space is required. */
+            S.AddFixedText(XO(" for more info."));
```

Comment:
This violates the “substitute, don’t concatenate” rule that I always recommend with internationalized strings.

Is wxHyperLinkedCtrl a technical necessity?  Then maybe we must break the rule here, but the i18n-hint comments  for the two surrounding strings should emphasize “translate this string pair as a whole” or similar.  Maybe Japanese wants to translate “See” in the second string.

Or maybe it could be handled like the recent change in the About dialog.

Reply:
I'm afraid - yes. I am actually doing the same changes to about dialog. HtmlWindow is not accessible and I'm not sure that it can be fixed in a reasonable time. This makes PP inaccessible for VI users.

Reply:
So then, be very clear with i18n hints in both places.